client/src/templates/Challenges/utils/index.js

```diff
@@ -1,7 +1,11 @@
-const guideBase = '/guide/certifications';
+const guideBase = 'https://www.freecodecamp.org/forum/search?q=';
```

Comment:
This logic will obviously change to the actual links later. I am just throwing it here for now, so that we have something to work with.